packages/salmon-wallet/__test__/send.test.ts

```diff
@@ -0,0 +1,186 @@
+import waitForExpect from 'wait-for-expect'
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { GenesisKeys } from '@defichain/testcontainers'
```

Comment:
```suggestion
import { RegTestGenesisKeys } from '@defichain/jellyfish-network'
```

always avoid deprecated functions, if we keep using them, it makes it hard to migrate away from them